application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -250,7 +259,7 @@ private static void handleFailure(Throwable exception, InteractionHook eventHook
     }
 
     private static RestAction<String> mentionHelpChangeCommand(Guild guild, String subcommand) {
-        return MessageUtils.mentionGuildSlashCommand(guild, COMMAND_NAME, CHANGE_SUBCOMMAND_GROUP,
-                subcommand);
+        return MessageUtils.mentionGuildSlashCommand(guild, HelpThreadCommand.COMMAND_NAME,
+                CHANGE_SUBCOMMAND_GROUP, subcommand);
```

Comment:
why did u prefix one but not the other? be consistent. either prefix both (and remove ur static import), or prefix none and roll with the static import

Reply:
@Zabuzard i can't use static import for `COMMAND_NAME` coz this class also have the same field with different value, so it'll use that one

Reply:
Good call by zabuzard. this just complicates things.

Reply:
@Taz03 thats fair, and indicates that a static import here just causes confusion. please get rid of it and fully classify both, thanks.

Reply:
done